docs/book/orders/checkout.rst

```diff
@@ -76,10 +76,26 @@ How to perform the Addressing Step programmatically?
 Firstly if the **Customer** is not yet set on the Order it will be assigned depending on the case:
 
 * An already logged in **User** - the Customer is set for the Order using the `CartBlamerListener <https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/CoreBundle/EventListener/CartBlamerListener.php>`_, that determines the user basing on the event.
-* An existent **User** that is not logged in - If there is an account in the system registered under the e-mail that has been provided - they are asked for a password to log in before continuing inside the addressing form.
-* A **Customer** that was present in the system before (we've got their e-mail) - the Customer instance is updated via cascade, the order is assigned to it.
+* A **Customer** or **User** that was present in the system before (we've got their e-mail) - the Customer instance is updated via cascade, the order is assigned to it.
 * A new **Customer** with unknown e-mail - a new Customer instance is created and assigned to the order.
 
+.. note::
+
+    Before Sylius ``v1.7`` a **User** (i.e. we have their e-mail and they are registered) had to login before they could complete the checkout process. If you want this constraint on your checkout, you can add this to your application:
```

Comment:
I wonder if we want to have a configuration checkbox on channel level about it? Similar to `Is account verification required?` cc @CoderMaggie

Reply:
I chatted with Maggie about that, and she said no, but it could be an option to have it.